client/state/reader/teams/actions.js

```diff
@@ -7,11 +7,26 @@
  */
 import {
 	READER_TEAMS_REQUEST,
+	READER_TEAMS_RECEIVE,
 } from 'state/action-types';
 
 export function requestTeams() {
 	return {
 		type: READER_TEAMS_REQUEST,
+		meta: {
+			requestStart: READER_TEAMS_REQUEST
+		}
+	};
+}
```

Comment:
Is there anything we can do here via convention or composition to make adding the dedupe meta easier?